kuma/wiki/jinja2/wiki/document.html

```diff
@@ -238,6 +238,10 @@
 
               <!-- just the article content -->
               <article id="wikiArticle">
+                {% if doc_abs_url == '/en-US/docs/Experiment:InteractiveEditor/Test.Array.prototype.reduce()' %}
+                  {{ add_interactive_editor() }}
```

Comment:
You need to import the macro on line 4 of document.html

Reply:
Ugh, had it in there and then had to revert formatting changes applied by prettier and neglected to add it back.